distribution/docs/src/main/resources/_contents/_draft-admin-contents/managing-admin-contents.adoc

```diff
@@ -0,0 +1,38 @@
+
+The ${ddf-admin} Application contains components that are responsible for the installation and configuration ${branding} applications.
```

Comment:
Nit, but it feels like anthropomorphizing to say the components are "responsible" for anything. Maybe "integral" or "key"?